ext/DataFramesExt.jl

```diff
@@ -0,0 +1,45 @@
+module DataFramesExt
+
+using NEOs
+import NEOs: RadecMPC, RadarJPL
+
+if isdefined(Base, :get_extension)
+    using DataFrames: DataFrame, nrow
+    import Tables: istable, rowaccess, rows, schema, Schema
+else
+    using ..DataFrames: DataFrame, nrow
+    import ..Tables: istable, rowaccess, rows, schema, Schema
+end
+
+# Methods to convert a Vector{RadecMPC{T}} to a DataFrame
+istable(::Type{Vector{RadecMPC{T}}}) where {T <: AbstractFloat} = true
+rowaccess(::Type{Vector{RadecMPC{T}}}) where {T <: AbstractFloat} = true
+rows(x::Vector{RadecMPC{T}}) where {T <: AbstractFloat} = x
+schema(::Vector{RadecMPC{T}}) where {T <: AbstractFloat} = Schema(fieldnames(RadecMPC{T}), Tuple{fieldtypes(RadecMPC{T})...})
+
+# Methods to convert a Vector{RadarJPL{T}} to a DataFrame
+istable(::Type{Vector{RadarJPL{T}}}) where {T <: AbstractFloat} = true
+rowaccess(::Type{Vector{RadarJPL{T}}}) where {T <: AbstractFloat} = true
+rows(x::Vector{RadarJPL{T}}) where {T <: AbstractFloat} = x
+schema(::Vector{RadarJPL{T}}) where {T <: AbstractFloat} = Schema(fieldnames(RadarJPL{T}), Tuple{fieldtypes(RadarJPL{T})...})
```

Comment:
It's great that the Tables.jl interface works so well, many thanks for taking care of this! I think we can go a step beyond and further simplify this to something like:
```suggestion
# Methods to convert a Vector{T} where {T <: AbstractAstrometry} to a DataFrame
istable(::Type{Vector{<: AbstractAstrometry}}) = true
rowaccess(::Type{Vector{<: AbstractAstrometry}}) = true
rows(x::Vector{<: AbstractAstrometry}) = x
schema(::Vector{T}) where {T <: AbstractAstrometry} = Schema(fieldnames(T), Tuple{fieldtypes(T)...})
```